dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java

```diff
@@ -184,8 +187,47 @@ private String normalizePath(String basePath, String path) {
             return path.startsWith("/") ? basePath + path : basePath + "/" + path;
         }
 
-        public List<String> getEndpoints() {
-            return Ordering.natural().sortedCopy(endpoints);
+        public List<EndpointLogLine> getEndpointLogLines() {
+            return endpointLogLines;
+        }
+    }
+
+    private static class EndpointLogLine {
+        private final String httpMethod;
+        private final String basePath;
+        private final Class<?> klass;
+
+        public EndpointLogLine(String httpMethod, String basePath, Class<?> klass) {
+            this.basePath = basePath;
+            this.klass = klass;
+            this.httpMethod = httpMethod;
+        }
+
+        public String getHttpMethod() {
```

Comment:
We could access these fields directly from the comparator, so these getters are not needed.